crates/core/src/commands/restore.rs

```diff
@@ -537,14 +555,7 @@ fn restore_contents<P: ProgressBars, S: Open>(
                                 let mut sizes_guard = sizes.lock().unwrap();
                                 let filesize = sizes_guard[file_idx];
                                 if filesize > 0 {
-                                    dest.set_length(path, filesize)
-                                        .map_err(|err| {
-                                            CommandErrorKind::ErrorSettingLength(
-                                                path.clone(),
-                                                Box::new(err),
-                                            )
-                                        })
-                                        .unwrap();
+                                    dest.set_length(path, filesize).unwrap();
```

Comment:
could this be the opportunity to remove these `unwrap`?

Reply:
I want to do that at a later stage, when we can focus on fixing the unwraps itself. This PR's focus should really be the RusticResult/RusticError, local errors and `map_err`, IMHO.